python/rpdk/java/templates/init/guided_aws/Translator.java

```diff
@@ -52,6 +52,7 @@ static ResourceModel translateFromReadResponse(final AwsResponse awsResponse) {
     // e.g. https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-logs/blob/2077c92299aeb9a68ae8f4418b5e932b12a8b186/aws-logs-loggroup/src/main/java/com/aws/logs/loggroup/Translator.java#L58-L73
     return ResourceModel.builder()
         //.someProperty(response.property())
+        // IMPORTANT: exclude writeOnlyProperties
```

Comment:
isn't this done by lambda wrapper?

Reply:
Yes it should be. At least in this language plugin.

Reply:
could you just remove this line about write only properties. This might cause some confusion.